tests/scripts/bootstrap.sh

```diff
@@ -133,10 +133,13 @@ case "$(uname)" in
         fi
 
         if [ "${OTBR_MDNS-}" == 'mDNSResponder' ]; then
-            SOURCE_NAME=mDNSResponder-878.30.4
+            SOURCE_NAME=mDNSResponder-1310.80.1
             wget https://opensource.apple.com/tarballs/mDNSResponder/$SOURCE_NAME.tar.gz \
                 && tar xvf $SOURCE_NAME.tar.gz \
-                && cd $SOURCE_NAME/mDNSPosix \
+                && cd $SOURCE_NAME/Clients \
+                && sed -i '/#include <ctype.h>/a #include <stdarg.h>' dns-sd.c \
+                && sed -i '/#include <ctype.h>/a #include <sys/param.h>' dns-sd.c \
```

Comment:
Is this a known issue of mDNSResponder? Do we have more context?

Reply:
We need to add these headers to import `MIN`, `va_*` macros, otherwise the build will fail. We've already applied the same fix in `script/bootstrap`.